### PR TITLE
Feat/add squeeth ref vol

### PR DESCRIPTION
**Pull request by DemolaJames**
# Task:

## Description

Added Squeeth Ref vol to main trade page 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

**Comment by DemolaJames** [comment]
Screenshot 2022-10-20 at 16 35 28

**Review by KMKoushik — APPROVED** [review]

**Comment by aleone** [comment]
@haythem96 I'm seeing 78% vol on the f/e, that is wrong, right?  Is it not updating in the db?

File: packages/frontend/src/constants/enums.ts [hunk 1]
```diff
@@ -82,6 +82,8 @@ export enum Tooltips {
   FundingMonthly = 'Monthly premium is calculated as (Daily Premium * 30)',
   FundingAnnual = 'Annual Premium is calculated as (Daily Premium * 365.25)',
   VaultCollatRatio = `You'll be adjusting the collateral ratio for the whole vault and not just this trade.`,
+  osqthRefVol = 'Squeeth reference volatility based on deribit options and squeeth replicating portfolio. Squeeth ref vol is calculated/refreshed every 5 minutes..Click the link icon to learn more',
```
Comment (alexisgauba) [inline packages/frontend/src/constants/enums.ts:85]:
```suggestion
  osqthRefVol = 'Squeeth reference volatility based on deribit options and squeeth replicating portfolio. Squeeth ref vol is calculated/refreshed every 5 minutes.',
```

Comment (alexisgauba) [reply packages/frontend/src/constants/enums.ts:85]: The link icon should be self-explanatory, if its not we need a better visual

File: packages/frontend/src/constants/index.ts [hunk 2]
```diff
@@ -46,3 +46,5 @@ export const V2_MIGRATION_OSQTH_PRICE = 0.0897
 export const V2_MIGRATION_ETH_AMOUNT = 790.186510787310292814
 export const V2_MIGRATION_OSQTH_AMOUNT = 4604.57349514942255453
 export const V2_MIGRATION_SUPPLY = 519.994374902152348633
+
+export const squeethRefVolDocLink = 'https://colab.research.google.com/drive/1HTM_2j0jmda9tzN_uskBPz9Rpma8Lp3C'
```
Comment (alexisgauba) [inline packages/frontend/src/constants/index.ts:50]: is this the right link or is there an update paper / medium @alpinechicken @southamerica ?

**Review by alexisgauba — CHANGES_REQUESTED** [review]

**Comment by alexisgauba** [comment]
I'm seeing 75.57% on the site but 64.9% on Joe's colab, looks like the FE is off? Even with the 5 min gap this seems quite off

Screen Shot 2022-10-21 at 9 02 20 PM

Screen Shot 2022-10-21 at 9 02 43 PM

**Comment by aleone** [comment]
The tooltip says 3 minutes, it is not 5 minutes, correct?  Lets make sure we have the correct refresh interval before merging.

**Review by alexisgauba — APPROVED** [review]

**Pull request merged**